stdlib/typing.pyi

```diff
@@ -591,7 +591,7 @@ class MutableMapping(Mapping[_KT, _VT], Generic[_KT, _VT]):
     @abstractmethod
     def __setitem__(self, __k: _KT, __v: _VT) -> None: ...
     @abstractmethod
-    def __delitem__(self, __v: _KT) -> None: ...
+    def __delitem__(self, __k: _KT) -> None: ...
```

Comment:
```suggestion
    def __delitem__(self, __key: _KT) -> None: ...
```

```pycon
>>> from collections.abc import MutableMapping
>>> help(MutableMapping.__delitem__)
Help on function __delitem__ in module collections.abc:

__delitem__(self, key)
```

Reply:
As far as I remember, `MutableMapping` and its arguments is very complex.
Because of how `dict` is implemented:

```python
>>> help(dict.__delitem__)
Help on wrapper_descriptor:

__delitem__(self, key, /)
    Delete self[key].
```

So, I prefer to not change any semantics here. Only a name that does not affect anything.

Reply:
Correct, we definitely shouldn't make the argument positional-or-keyword! That would break a lot of things :)